physionet-django/tests.py

```diff
@@ -0,0 +1,67 @@
+from schema import Query
```

Comment:
For consistency with other tests, should this go in the app subfolder, rather than the root folder?

Reply:
So are you saying should the top-level `schema.py` file go into the app subfolder? If so, each app already has its own `schema.py` (or will after more than just the ones in `export` are added) and there should also be a project-level `schema.py` to wrap them all together. Do you think testing should be done for each app instead of just one testing file for all the QraphQL schemas together? Maybe only the `export` app folder should have the schemas and queries and tests?

Reply:
sorry, it wasn't clear, but the comment doesn't refer to the import but to the module itself. most tests are in the associated app folder, but this test is in the project root. for consistency, perhaps move the test to `export` (which isn't a good name, but that's another story!).